src/main/java/org/gridsuite/casevalidation/server/CaseValidationService.java

```diff
@@ -39,12 +48,31 @@ private Network getNetwork(UUID networkUuid) {
         }
     }
 
-    CaseValidationReport validate(UUID networkUuid, LoadFlowParameters parameters) {
+    CaseValidationReport validate(UUID networkUuid) {
         Network network = getNetwork(networkUuid);
-        LoadFlowParameters params = parameters != null ? parameters : new LoadFlowParameters();
+        List<LoadFlowCaseValidationReport> loadFlowReports = new ArrayList<>();
+
+        //Validation with default loadflow parameters
+        LoadFlowParameters params = new LoadFlowParameters()
```

Comment:
All this code related to load flow based validation, should be move in LoadFlowCaseValidationReport

Reply:
Done